C/services/north/include/data_load.h

```diff
@@ -61,6 +61,80 @@ class DataLoad : public ServiceHandler {
 						m_prefetchLimit = limit;
 					};
 
+		// Debugger entry points
```

Comment:
Since the implementations are in the include file, they will be inlined. Is that intended?

Reply:
Yes, it was